### ISSUE TITLE
Calendar - Ensure elements with an ARIA role that require parent roles are contained by them

### ISSUE THREAD

**Issue by jcfilben** [grommet/grommet#7424]
### Expected Behavior
Calendar should pass all automated accessibility checks

### Actual Behavior
Calendar has the following accessibility error: 'Ensure elements with an ARIA role that require parent roles are contained by them'
An element in Calendar has `role="row"` and doesn't have the correct parent role.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/row_role

https://dequeuniversity.com/rules/axe/4.10/aria-required-parent?application=axeAPI

### URL, screen shot, or Codepen exhibiting the issue
https://storybook.grommet.io/?path=/story/visualizations-calendar-simple--simple

### Steps to Reproduce

1. 2. 3.

### Your Environment

- Grommet version:
- Browser Name and version:
- Operating System and version (desktop or mobile):
